### PR TITLE
Doc: Improve man page for abrt-action-analyze-vulnerability

### PR DESCRIPTION
**Pull request by mzidek-gh**
Add more infromation about the tool and exploitable rating
in general.

Related:
https://github.com/abrt/abrt/issues/665

### CODE REVIEW

**Comment by mzidek-gh** [comment]
NOTE: We are most likely going to deprecate this tool in the future. In it's current form it is very limited and a lot of behaviour is hard-coded (like ignoring lower exploitable ratings). When a new tool is introduced it will come with it's own documentation.

File: doc/abrt-action-analyze-vulnerability.txt [hunk 1]
```diff
@@ -13,12 +13,26 @@ DESCRIPTION
...
 
+EXPLOITABLE RATING
+------------------
+Exploitable rating is a score (on scale 0-9) given to a coredump based on
+the likely reason the program crashed and whether or not it could have been due
+to a potentially exploitable security issue. Lower numbers are reserverd for
+reasons that are highly unlikely to be security issues (0-3) while
+higher numbers (4-9) are reserved for potentially dangerous crashes. Higher
+scores do not automatically mean security problems but they can help with
+debugging the issue and assessing it's severity.
```
Comment (mgrabovsky) [inline doc/abrt-action-analyze-vulnerability.txt:34]:
```suggestion
debugging the issue and assessing its severity.
```

Comment (mzidek-gh) [reply doc/abrt-action-analyze-vulnerability.txt]: Thanks for the catch. Fixed.

**Comment by mzidek-gh** [comment]
Updated.

**Review by mgrabovsky — APPROVED** [review]
Looks good.

**Pull request merged**